aapp_runner/tle_satpos_prepare.py

```diff
@@ -178,6 +182,33 @@ def download_tle(config, timestamp, dir_data_tle):
     return tle_file_list
 
 
+def fetch_realtime_tles(tle_input_path, tle_output_path, tle_infile_format):
```

Comment:
This function needs to be tested.

Reply:
I have added a test now. Let me hear what you think?